src/view/com/post-thread/PostThreadItem.tsx

```diff
@@ -325,12 +325,6 @@ let PostThreadItemLoaded = ({
             {post.repostCount !== 0 || post.likeCount !== 0 ? (
               // Show this section unless we're *sure* it has no engagement.
               <View style={[styles.expandedInfo, pal.border]}>
-                {post.repostCount == null && post.likeCount == null && (
```

Comment:
Can this still be the case if you click through a nested chain of embeds really fast? If yes, might want to leave this in.